mlog/log.go

```diff
@@ -109,6 +124,24 @@ func NewLogger(config *LoggerConfiguration) *Logger {
 	return logger
 }
 
+func NewLogProxy(tb testing.TB) *Logger {
```

Comment:
Can you write a godoc for this function? Also, suggest calling this a `NewTestingLogger`.

Reply:
Sure! :smile: